sources/packages/backend/apps/api/src/route-controllers/student/student.students.controller.ts

```diff
@@ -331,7 +331,7 @@ export class StudentStudentsController extends BaseController {
         "Not able to update a student due to an invalid request.",
       );
     }
-    await this.studentService.updateStudentContactByStudentId(
+    await this.studentService.updateStudentInfoStudentId(
```

Comment:
I guess that there is a "by" missing `updateStudentInfoByStudentId`. Either way `updateStudentInfo` would be ok IMO also.